spec/features/accessibility/idv_pages_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 require 'axe/rspec'
 
-feature 'Accessibility on IDV pages', :js do
+feature 'Accessibility on IDV pages', :js, :accessibility do
```

Comment:
What about changing the command line arguments to add `--exclude-pattern "accessibility"`. That way as long as we keep the tests in the right directory we don't have to worry about tagging the tests?